resources/ext.wikibase.facetedsearch.js

```diff
@@ -0,0 +1,78 @@
+let input;
```

Comment:
This is a bit generic. Same with `toggle` elsewhere in recently added code. There are also technical terms that say nothing about the domain.

`searchInput` is clearer

Scope also matters. For variable names or constants, the bigger the scope, the bigger the name. So inside of a short function, `input` might be optimal, but for this case we have something used in the entire file.

Reply:
I agree that we should have it more specific.
I'm not sure about `searchInput` because it is associated with search box (`#searchInput`) provided by MediaWiki skins.
What do you think about a term such as `specialSearchInput`?

Reply:
I got onto `searchInput` because the selector is `#searchText > input`

I have some aversion to  `specialSearchInput` due to `special`. At least it isn't magic :) 

From the context it probably is clear what "search input" we refer to, so specifying it is the one added by the special page might be overly specific.

_Bikeshedding alert, spending too much time on this_